### PR TITLE
update redis to valkey in generate-commands-json.py

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
Previously these scripts were updated but still some places are left so updated the valkey.

### CODE REVIEW

File: utils/generate-command-code.py [hunk 1]
```diff
@@ -576,7 +576,7 @@ def create_command(name, desc):
 """
 /* We have fabulous commands from
  * the fantastic
- * Redis Command Table! */
+ * Server Command Table! */
```
Comment (madolson) [inline utils/generate-command-code.py]: You need to re-generate the commands table, since you are updating this comment.

**Comment by Shivshankar-Reddy** [comment]
Hi @madolson , Resolved the conflicts and somebody already updated gen-command-code.py, Now only json file's changes to be reviewed, Can you please look it ? Thanks!

**Comment by Shivshankar-Reddy** [comment]
@zuiderkwast Could you please have a look at this when you have some time?

**Review by zuiderkwast — APPROVED** [review]
OK, good!

`generate-commands-json` doesn't seem to be used automatically in any CI jobs. The only reference I can find to it is in src/commands/README.md.

**Pull request merged**